tracer/test/test-applications/integrations/Samples.MSTestTests/Samples.MSTestTests.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <IsPackable>false</IsPackable>
     <OutputType>Library</OutputType>
-    <ApiVersion Condition="'$(ApiVersion)' == ''">3.8.1</ApiVersion>
+    <ApiVersion Condition="'$(ApiVersion)' == ''">3.9.1</ApiVersion>
```

Comment:
👍 I like that you did this, we should probably do it for all our Samples tbh, to make sure we're testing latest on .NET FX